test/unittests/module_test.cpp

```diff
@@ -34,7 +34,7 @@ TEST(module, functions)
     EXPECT_EQ(module->get_code(1).local_count, 0);
     EXPECT_EQ(module->get_code(2).instructions.size(), 1);
     EXPECT_EQ(module->get_code(2).local_count, 1);
-    EXPECT_EQ(module->get_code(3).instructions.size(), 2);
+    EXPECT_EQ(module->get_code(3).instructions.size(), 6);
```

Comment:
Unfortunately this is now rather misleading, as prior it told us there were 2 instructions.

Reply:
What do you propose?

Reply:
Nothing we can really do here.